_includes/footer.html

```diff
@@ -34,9 +34,12 @@
           alt="[Solid logo]"
         />
       </a>
-      <a class="navbar-item" href="mailto:info@solidproject.org">
+      <a class="navbar-item is-hidden-mobile" href="mailto:info@solidproject.org">
         info@solidproject.org
       </a>
+      <a class="navbar-item is-size-3-mobile is-hidden-tablet" href="mailto:info@solidproject.org">
```

Comment:
The `@` looks good to me, but it might be confusing to people with visual impairments. If you add this, a screen reader should be able to read it properly to them:

```suggestion
      <a class="navbar-item is-size-3-mobile is-hidden-tablet" href="mailto:info@solidproject.org" aria-label="info@solidproject.org">
```